src/WingmanDataTrail/MetricVizPanel/MetricVizPanel.tsx

```diff
@@ -111,6 +113,28 @@ export class MetricVizPanel extends SceneObjectBase<MetricVizPanelState> {
         .build();
     }
 
+    // check if metric is histogram, if yes build heatmap panel
+    const isHistogram = metricName.endsWith('_bucket');
```

Comment:
We still need to show support for native histograms 😄 

is it possible to access the data source helper to get access to this code so we can check for histograms that do not follow the `_bucket` naming convention?

https://github.com/grafana/metrics-drilldown/blob/98b18ba52971f4c6bc2d51e40eef4a8d3a1505a3/src/MetricSelect/MetricSelectScene.tsx#L447

If you can get access to the parent trail, you will be able to pass the metric name to this function. It will tell you if a metric is a native histogram or not